src/main/java/db/migration/CommonDbUtils.java

```diff
@@ -0,0 +1,23 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class CommonDbUtils {
+
+    public static void insertFile(Context context, String name, String file) throws IOException {
```

Comment:
Depending on needs of table that we're actually inserting to, name this method more appropriately and pass in relevant args